File: backend/src/main/java/com/socialmeet/backend/dto/AlipayOrderResponse.java
package com.socialmeet.backend.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

/**
 * 支付宝订单响应DTO
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class AlipayOrderResponse {
    
    private String orderId;
    private String alipayOrderInfo;
    private String alipayOutTradeNo;
    private String qrCode;
    private String payUrl;
    private Long expireTime;
}

File: app/src/main/java/com/example/myapplication/model/AlipayOrderResponse.kt
package com.example.myapplication.model

/**
 * 支付宝订单响应数据类
 */
data class AlipayOrderResponse(
    val orderId: String,
    val alipayOrderInfo: String,
    val alipayOutTradeNo: String,
    val qrCode: String? = null,
    val payUrl: String? = null,
    val expireTime: Long
)
